src/thread_rng.rs

```diff
@@ -166,8 +163,7 @@ impl CryptoRng for ThreadRng {}
 /// ```
 ///
 /// [`thread_rng`]: fn.thread_rng.html
-/// [`Rand`]: trait.Rand.html
-#[deprecated(since="0.5.0", note="removed in favor of thread_rng().gen()")]
+/// [`Standard`]: distributions/struct.Standard.html
```

Comment:
Maybe we should mention the disadvantages over using `thread_rng().gen()` here.

Reply:
I don't think this is needed — there's already a note about caching the `thread_rng` handle above